training/a4/llama3-1-70b/nemo-pretraining-gke/16node-bf16-seq8192-gbs512-gcs/README.md

```diff
@@ -0,0 +1,284 @@
+    --set volumes.gcsMounts[0].bucketName=${GCS_BUCKET_LOGS} \
+    --set volumes.gcsMounts[0].mountPath=/job-logs \
+    --set workload.enable_dataloading=$ENABLE_DATALOADING \
+    --set workload.enable_ckpt_write=$ENABLE_CHECKPOINT_WRITE \
+    --set workload.enable_ckpt_load=$ENABLE_CHECKPOINT_LOAD \
+    --set workload.ckpt_write_interval=$CHECKPOINT_WRITE_INTERVAL \
+    --set workload.token_path=$TOKEN_PATH \
+    --set workload.dataset_path=$DATASET_PATH \
+    --set workload.restore_path=$RESTORE_PATH 
+    ```
+
+### Monitor the job
+
+To check the status of pods in your job, run the following command:
+
+```
+kubectl get pods | grep a4-llama3-1-70b-gpus128
+```
+
+Replace the following:
+
+- JOB_NAME_PREFIX - your job name prefix. For example a4-llama3-1-70b-gpus128.
+
+To get the logs for one of the pods, run the following command:
+
+```
+kubectl logs POD_NAME
+```
+
+Information about the training job's progress, including crucial details such as
+loss, step count, and step time, is generated by the rank 0 process.
+This process runs on the pod whose name begins with
+`JOB_NAME_PREFIX-workload-0-0`.
+For example: `a4-llama3-1-70b-gpus128-workload-0-0-s9zrv`.
+
+### Analyze results
+
+When completed, the job creates several artifacts, including logs and traces, and places them
+in the  Google Cloud Storage logs bucket as follows:
+
+```
+gs://${GCS_BUCKET_LOGS}/nemo-experiments-storage/<JOB_ID>
+├── nemo-configuration.yaml
+├── lightning_logs.txt
+├── nemo_error_logs.txt
+├── nemo_log_globalrank-[RANK]_localrank-[LOCAL].txt
+├── dllogger
+│   ├── rank-0
+│   │   ├── dllogger.json
+...
+```
+
+- `nemo-configuration.yaml`: the NeMo configuration used by the pretraining script. This includes
+   the combined [configuration file](../16node-bf16-seq8192-gbs512/llama3-1-70b.py)
+   and the command line overrides
+- `lightning_logs.txt`: the log files generated by PyTorch Lightning, which is used by NeMo
+- `nemo_error_logs.txt`: the warning and error logs generated by NeMo
+- `nemo_log_globalrank-[RANK]_localrank-[LOCAL].txt`: the NeMo logs for each rank
+- `dllogger/`: The log captured by [NVIDIA DLLogger](https://github.com/NVIDIA/dllogger):
+   DLLogger is configured to store logs on the rank 0 node. The log is in JSON format
+   and includes loss, step_time, and other key metrics for each training step
+
+The `<JOB_ID>` has the following format:
+- `$USER--llama31-70b-gcs-[YYYY]-[MM]-[DD]-[hh]-[mm]-[ss]`, where the suffix of the ID is a day and time when the job was started.
+
+
+The NeMo log files include information about checkpoint operations on each rank. You can use the [checkpointing_metrics](../../../../src/utils/checkpointint_metrics) utility to calculate statistics for checkpoint write times.
+
+To calculate statistics:
+
+
+1. Set a path to the NeMo logs.
+
+```
+export JOB_ID=<JOB_ID>
+export GCS_LOGS_PATH="gs://${GCS_BUCKET_LOGS}/nemo-experiments-storage/${JOB_ID}"
+```
+
+Replace `<JOB_ID>` with the ID of your job.
```

Comment:
This section seems to end abruptly. I'm ok if we don't want to update the checkpoint metrics utility, but we should at least tell users where they can find this data in the logs.